app/renderer/actions/Session.js

```diff
@@ -481,6 +481,7 @@ export function newSession (caps, attachSessId = null) {
       protocol: https ? 'https' : 'http',
       path,
       connectionRetryCount: CONN_RETRIES,
+      connectionRetryTimeout: 10 * 60 * 1000,
```

Comment:
let's put magic constants up top with a descriptive variable name.

and we're sure we want to make it a 10 minute connection timeout in all cases, not just the cloud provider case? not sure this makes sense for local servers.